### PR TITLE
Set number of workers with venv

### PR DESCRIPTION
**Pull request by matrohon**
Signed-off-by: Mathieu Rohon <mathieu.rohon@orange.com>

## Description

The WORKER_COUNT can be set to control the number of k8s workers to deploy with vagrant.

## Motivation and Context

We can then easily set the number of worker to 0 for development.

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [x] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .mk/vagrant.mk [hunk 1]
```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+
```
Comment (nickolaev) [inline .mk/vagrant.mk]: Maybe we do not want these empty lines?

File: .mk/vagrant.mk [hunk 2]
```diff
@@ -36,22 +38,29 @@ vagrant-resume:
...
+vagrant-ssh-worker%:
+	@cd scripts/vagrant; vagrant ssh worker$*
 
-.PHONY: vagrant-kublet-restart
-vagrant-restart-kublet:
-	@cd scripts/vagrant; vagrant ssh master -c "sudo service kubelet restart"; vagrant ssh worker -c "sudo service kubelet restart"
+.PHONY: vagrant-restart-kubelet
+vagrant-restart-kubelet:
+	@cd scripts/vagrant; vagrant ssh master -c "sudo service kubelet restart"; \
+	number=1 ; while [[ $$number -le ${WORKER_COUNT} ]] ; do \
```
Comment (nickolaev) [inline .mk/vagrant.mk]: Coud this be `for`, mabe a bit more obvious?

File: docs/guide-kind.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 [`kind`](kind.sigs.k8s.io) is a tool for running local Kubernetes clusters using Docker container “nodes”.
 Docker is the only prerequisite, it does not require any additional steps, hypervisors etc.
 
-It is worth noting that `kind` as any other Kubernetes deplyment tool would expect that the machine that hostst the Docker has at lease 4 CPU cores and 4 GB of RAM. That is specifically pointed for OSX users in the official [docs](https://kind.sigs.k8s.io/docs/user/quick-start/).
+It is worth noting that `kind` as any other Kubernetes deployment tool would expect that the machine that hosts the Docker has at least 4 CPU cores and 4 GB of RAM. That is specifically pointed for OSX users in the official [docs](https://kind.sigs.k8s.io/docs/user/quick-start/).
```
Comment (nickolaev) [inline docs/guide-kind.md]: That does not belong to this PR.

File: .mk/vagrant.mk (hunk 2 above)
Comment (matrohon) [reply .mk/vagrant.mk]: while or until loop seems more adapted than a "for in" loop: 
http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO-7.html

no?
